cmd/flags.go

```diff
@@ -130,6 +136,14 @@ Pattern: %q`, 1, 7, "^([1-9][0-9]*|[0]|[1-9][0-9]%|[0-9]%|100%)$")
 	return maxErrors, nil
 }
 
+func validateSessionFlags(cmd *cobra.Command, instanceList []string, filterList map[string]string) error {
```

Comment:
It's either this or just `--instance` would get priority and `--filter` would be ignored.

Reply:
I'd prefer to be explicit about it (as we are with validateRunFlags) so that users have a clear understanding of how the flags work. The reason we're making it fail is because passing instance IDs as well as filters will cause the AWS API to throw an error, and I think we should reflect that here.